camera_data.py

```diff
@@ -5,8 +5,8 @@
 
 class CameraData:
     def __init__(self, camera_tilt):
-        self.H_FIELD_OF_VIEW = 68.37
-        self.V_FIELD_OF_VIEW = 41.21
+        self.H_FIELD_OF_VIEW = 72.96
```

Comment:
Let's rename this to be more descriptive of what is actually represented by this value, and also give the units.

Reply:
Thank you for the suggestion. It has been updated.